testing/dart/color_test.dart

```diff
@@ -339,6 +339,11 @@ void main() {
     // Call base class member, make sure it uses overridden value.
     expect(color.red, 0xE0);
   });
+
+  test('value and toARGB32 are equivalent methods', () {
+    const Color halfRed = Color.from(alpha: 1.0, red: 0.5, green: 0, blue: 0);
+    expect(halfRed.value, halfRed.toARGB32());
```

Comment:
It's probably better not to introduce a new test that uses a deprecated property.  I'd just assert it is a specific integer.

Reply:
Sounds good!